[2020-06-03 15:48:03,223] {taskinstance.py:669} INFO - Dependencies all met for <TaskInstance: covid_dag.source_to_csv 2020-06-03T10:13:51.956291+00:00 [queued]>
[2020-06-03 15:48:03,229] {taskinstance.py:669} INFO - Dependencies all met for <TaskInstance: covid_dag.source_to_csv 2020-06-03T10:13:51.956291+00:00 [queued]>
[2020-06-03 15:48:03,229] {taskinstance.py:879} INFO - 
--------------------------------------------------------------------------------
[2020-06-03 15:48:03,229] {taskinstance.py:880} INFO - Starting attempt 4 of 4
[2020-06-03 15:48:03,229] {taskinstance.py:881} INFO - 
--------------------------------------------------------------------------------
[2020-06-03 15:48:03,237] {taskinstance.py:900} INFO - Executing <Task(PythonOperator): source_to_csv> on 2020-06-03T10:13:51.956291+00:00
[2020-06-03 15:48:03,239] {standard_task_runner.py:53} INFO - Started process 17218 to run task
[2020-06-03 15:48:03,275] {logging_mixin.py:112} INFO - Running %s on host %s <TaskInstance: covid_dag.source_to_csv 2020-06-03T10:13:51.956291+00:00 [running]> nineleaps-ThinkPad-L470
[2020-06-03 15:48:04,146] {python_operator.py:114} INFO - Done. Returned value was: None
[2020-06-03 15:48:04,150] {taskinstance.py:1065} INFO - Marking task as SUCCESS.dag_id=covid_dag, task_id=source_to_csv, execution_date=20200603T101351, start_date=20200603T101803, end_date=20200603T101804
[2020-06-03 15:48:13,227] {logging_mixin.py:112} INFO - [2020-06-03 15:48:13,227] {local_task_job.py:103} INFO - Task exited with return code 0
